manifests/init.pp

```diff
@@ -17,6 +17,7 @@
 # @param provider What provider to use for installation of the packages, except gunicorn and Python itself.
 # @param use_epel to determine if the epel class is used.
 # @param manage_scl Whether to manage core SCL packages or not.
+# @param umask The default umask for invoked exec calls.
```

Comment:
Needs to update `REFERENCE.md` with following command :
`puppet strings generate --format markdown --out REFERENCE.md`

Reply:
Done.